### ISSUE TITLE
Documentation on racket xp mode mentions annotations about tail position which were removed

### ISSUE THREAD

**Issue by greenfork** [greghendershott/racket-mode#524]
https://www.racket-mode.com/#racket_002dxp_002dmode has this point:

Visually annotating expressions in a tail position, as well as the enclosing expression with respect to which they are in a tail position.

Since this feature was removed in https://github.com/greghendershott/racket-mode/commit/48084a44e50a3ecd91e40ac5b58eed297e314f3f, maybe it is worth to delete it or replace with something else

**Comment by greghendershott** [comment]
The history on this is:

- Added tail positions highlighted using two customizable faces.
- Added the text annotations.
- Added commands to navigate among the tail and context positions.
- Got feedback they were too distracting; removed them.
- Refreshed the doc string for `racket-xp-mode`.
- Forgot to `make deploy` the docs to refresh racket-mode.com.
- But I did just now. The web site docs now match the built-in doc string:

> When point is on the opening parenthesis of an expression in tail position, it is highlighted using the face racket-xp-tail-position-face.
>
> When point is on the opening parenthesis of an enclosing expression with respect to which one or more expressions are in tail position, it is highlighted using the face racket-xp-tail-target-face.

TL;DR: Thanks for pointing out the web site wasn't refreshed; done.
